src/main/java/com/g4s8/ghman/user/PgThreads.java

```diff
@@ -37,7 +37,7 @@
  * @since 1.0
  * @checkstyle MagicNumberCheck (500 lines)
```

Comment:
@victornoel I'd suggest to move this checkstyle suppression to methods scope, it seems to be a good practice to make suppressions work for as small peace of code as possible